This patch fixes Moodle installation with MariaDB 5.5.63

--- a/lib/dml/mysqli_native_moodle_database.php   2021-05-15 19:10:59.000000000 +0300
+++ b/lib/dml/mysqli_native_moodle_database.php   2021-05-27 09:31:29.813398829 +0300
@@ -282,7 +282,7 @@
         if (!$collation) {
             // We want only utf8 compatible collations.
             $collation = null;
-            $sql = "SHOW COLLATION WHERE Collation LIKE 'utf8mb4\_%' AND Charset = 'utf8mb4'";
+            $sql = "SHOW COLLATION WHERE Collation LIKE 'utf8\_%' AND Charset = 'utf8'";
             $result = $this->mysqli->query($sql);
             while ($res = $result->fetch_assoc()) {
                 $collation = $res['Collation'];
