This patch fixes Moodle installation with MariaDB 5.5.63

--- a/lib/dml/mysqli_native_moodle_database.php 2020-05-18 15:19:56.976492874 +0300
+++ b/lib/dml/mysqli_native_moodle_database.php 2020-05-18 15:20:08.284523752 +0300
@@ -274,7 +274,7 @@
         if (!$collation) {
             // We want only utf8 compatible collations.
             $collation = null;
-            $sql = "SHOW COLLATION WHERE Collation LIKE 'utf8mb4\_%' AND Charset = 'utf8mb4'";
+            $sql = "SHOW COLLATION WHERE Collation LIKE 'utf8\_%' AND Charset = 'utf8'";
             $this->query_start($sql, NULL, SQL_QUERY_AUX);
             $result = $this->mysqli->query($sql);
             $this->query_end($result);

